app/swagger/schemas/address.rb

```diff
@@ -10,20 +10,33 @@ class Address
         property :data, type: :object do
           key :required, [:attributes]
           property :attributes, type: :object do
-            property :type, type: :string, enum:
-              %w(
-                DOMESTIC
-                INTERNATIONAL
-                MILITARY
-              ), example: 'DOMESTIC'
-            property :address_effective_date, type: :string, example: '1973-01-01T05:00:00.000+00:00'
-            property :address_one, type: :string, example: '140 Rock Creek Church Rd NW'
-            property :address_two, type: :string, example: ''
-            property :address_three, type: :string, example: ''
-            property :city, type: :string, example: 'Washington'
-            property :state_code, type: :string, example: 'DC'
-            property :zip_code, type: :string, example: '20011'
-            property :zip_suffix, type: :string, example: '1865'
+            property :address do
+              property :type, type: :string, enum:
+                %w(
+                  DOMESTIC
+                  INTERNATIONAL
+                  MILITARY
+                ), example: 'DOMESTIC'
+              property :address_effective_date, type: :string, example: '1973-01-01T05:00:00.000+00:00'
+              property :address_one, type: :string, example: '140 Rock Creek Church Rd NW'
+              property :address_two, type: :string, example: ''
+              property :address_three, type: :string, example: ''
```

Comment:
For my taste, including the parent name in the child attribute doesn't read well (e.g. address.**address**_one).  I'd recommend:

```
address
  effective_date
  line_one
  line_two
  line_three
```

This is just my subjective (and probably late) opinion that really isn't relevant to this PR, so totally ignorable.

Reply:
Yeah, agree but since these properties are all passed through from EVSS APIs, it's somewhat easier to keep track of things if we don't mangle the names. For future EVSS APIs, we should try to suggest better property names when we see the opportunity.